examples/delay-pod.js

```diff
@@ -0,0 +1,81 @@
+import { sleep } from 'k6';
+import { Kubernetes } from 'k6/x/kubernetes';
+import { PodDisruptor } from '../src/pod.js';
+import { DeploymentHelper } from '../src/helpers.js';
+import  http from 'k6/http';
+
+const app = "httpbin"
+const image = "kennethreitz/httpbin"
+
+export function setup() {
+  const k8sClient = new Kubernetes()
```

Comment:
I know why its required, but my mind crashes a bit when we declare a new client in each code block 😄

Reply:
Maybe I should define it in the `init` code?